src/noun.time

```diff
@@ -238,39 +244,11 @@ d: the first of the four Sundays during Advent
 hp: Advent[1]
 hypo: Christian_holy_day
 
-w: Age_of_Fishes
```

Comment:
I don't like the idea of removing this. See wikipedia: "Fish reached substantial diversity during this time, leading the Devonian to often be dubbed the Age of Fishes"

I would not remove the lexical entries (Age of Fishes) nor the additional observataions in the glosses.

Reply:
same for all other cases where you remove MWEs and reduce glosses of `popular` names for the periods.

Reply:
Solved for all cases in last commit.

Checked all the information in the previous/original glosses against wikipedia. Somo were correct (and brought back), some were incorrect (and kept out) and some incomplete (and improved).

Also brought back the other names (synonyms) where they Wikipedia validation was available.